tests/unit/applications/test_emails.py

```diff
@@ -5,11 +5,18 @@
 
 def test_send_finished_notification(mailoutbox):
     application = ApplicationFactory()
+    application.submitted_by = application.created_by
 
     send_submitted_application_email("test@example.com", application)
 
     m = mailoutbox[0]
-
-    assert application.get_absolute_url() in m.body
+    text_content = m.body
+    html_content = m.alternatives[0][0]
 
     assert list(m.to) == ["test@example.com"]
+    assert application.get_absolute_url() in text_content
+    assert application.get_absolute_url() in html_content
+    assert application.submitted_by.name in text_content
+    assert application.submitted_by.name in html_content
+    assert f"ref: {application.pk_hash}" in text_content
+    assert f"ref: {application.pk_hash}" in html_content
```

Comment:
I think this is reasonable. We want to test that the text/HTML contains the most important information; we don't want to test where that information is located or how that information is formatted.